client/client.go

```diff
@@ -1,44 +1,23 @@
 package client
 
 import (
-	"io"
-
 	"github.com/nspcc-dev/neofs-api-go/v2/rpc/client"
 )
 
-// Client represents NeoFS client.
-type Client interface {
-	Accounting
-	Container
-	Netmap
-	Object
-	Session
-	Reputation
-
-	// Raw must return underlying raw protobuf client.
-	Raw() *client.Client
-
-	// Conn must return underlying connection.
-	//
-	// Must return a non-nil result after the first RPC call
-	// completed without a connection error.
-	Conn() io.Closer
-}
-
-type clientImpl struct {
+type Client struct {
```

Comment:
Should we save some describing comment?

Reply:
sure, i wanted to add a description after #91 when it is clearer how to handle errors from API calls

added desc in that PR